src/index.mjs

```diff
@@ -190,13 +212,71 @@ async function processCondaEnvironment(filePath) {
     }
 }
 
+// Fetch modified lines in the PR
+async function getModifiedLines(filePath) {
```

Comment:
can you move this to its own file as per https://github.com/openteamsinc/actions/pull/8#discussion_r1775771704